techmatters/flex-plugins
# ServiceConfig - 2 - CHI-2059 - Local json config sync migration

Primary reviewer:

## Description

This is the initial PoC of the remote config sync system that attempts to create all of the local json files. It looks enormous, but you can safely ignore all of the new json in the `twilio-iac/helplines` directory. They are updated again after some fixes in #1535 anyway.

### Checklist

### Related Issues
Fixes #....

### Verification steps

Don't verify anything here checkout the branch in the lowest level of the PR chain and verify there.
